translations/base-pt-BR.yaml

```
@@ -527,31 +527,31 @@ buildings:
         levelShortcut: LVL
     wire:
         default:
-            name: Energy Wire
-            description: Allows you to transport energy.
+            name: &wire Cabo de Energia
+            description: Permite transportar energia.
     advanced_processor:
         default:
-            name: Color Inverter
-            description: Accepts a color or shape and inverts it.
+            name: Inversor de Cor
```

The `&advanced_processor` anchor is missing here. It is aliased from line 806.
```suggestion
            name: &advanced_processor Inversor de Cor
```